packages/api-openfin/notification.html

```diff
@@ -0,0 +1,11 @@
+<h5 id="message">Message</h5>
```

Comment:
Ah, I see - when using dev tools, you're not seeing the original HTML source of the page which has been loaded - it's showing the DOM, the browsers interpretation of the HTML.

The browser will try to add missing elements, correct invalid markup etc. Therefore the original HTML can differ from the DOM shown in the dev tools. I don't OpenFin is injecting the template into the body here, I think it's just the browser.

It's still very minor, as the browser can correct it, but I think it's worth ensuring the HTML is valid, with doctype et. al.